msm8996.c

```diff
@@ -86,6 +94,10 @@ static struct debug_mux apc0_mux = {
 	.phys = 0x06400000,
 	.size = 0x1000,
 
+	.measure = measure_leaf,
+	.parent = &gcc.mux,
+	.parent_mux_val = 0x00bb,
```

Comment:
Question for the original code: why not just `0xbb`?

Reply:
Just because the table around it was using 4-difit values for the mux value. Fixed this chunk now.